README.md

```diff
@@ -29,7 +29,18 @@ steps:
     dotnet-version: '3.1.x' # SDK Version to use; x will use the latest version of the 3.1 channel
 - run: dotnet build <my project>
 ```
-
+Multiple versions:
+```yml
+steps:
```

Comment:
I think we need to clarify which version will be used by default if we install multiple versions. Maybe something like this:
```
Note: In case multiple versions are installed, the latest .NET version will be used by default unless another version is specified in the `global.json` file.
```

cc @vsafonkin @dmitry-shibanov